content/ar/courses/350/intro/python.md

```diff
@@ -1,87 +1,86 @@
 ---
 # Page title
-title: Python Review
+title: مراجعة البايثون
 
 # Title for the menu link if you wish to use a shorter link title, otherwise remove this option.
-linktitle: Python
+linktitle: باثون
 
 # Date page published
-date: 2021-03-23
+date: ٢٣-٣-٢٠٢١
 
 # Academic page type (do not modify).
-type: book
+type: كتاب
 
 # Position of this page in the menu. Remove this option to sort alphabetically.
-weight: 2
+weight: ٢
 
-draft: False
+draft: لا يوجد
```

Comment:
@Yousif-ALF well done .. please fix the following

type, weight, draft and type should not be translated. Please return them back to english to I can merge